### DISCUSSION TITLE
SumatraPDF in Microsoft Store - paid?

### DISCUSSION THREAD

**Discussion by okolnost** [sumatrapdfreader/sumatrapdf#3135] [General]
Just wanted to confirm that you are aware that SumatraPDF can be bought in Microsoft Store for 5 EUR.
![image](https://user-images.githubusercontent.com/690137/204000446-5069043d-4392-40ec-88e6-5aaf2717a883.png)

I do not see a way how to verify the publisher there - or affiliation to this project.
All that is there is just a publisher info...
![image](https://user-images.githubusercontent.com/690137/204001042-608e40db-6e8e-40fa-93a3-07c7d15b1886.png)

**Comment by GitHubRulesOK** [comment]
Very poor privacy notice that data is not used by Apple !!! LOL
clearly a pirate but not as cheeky as Chinese charge over $100 that claim they sell by the millions.

@kjk I think you need to publish the official version to stop such copymongers

**Comment by HugoHugoson** [comment]
Documenting another instance:
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/17321842/af730682-3471-4419-848a-8464bb7114b3)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/17321842/7645a8fe-ecd6-4ba8-8ffa-fe3738d2f2ca)

**Reply by GitHubRulesOK** [reply]
hmm 

https://apps.microsoft.com/store/detail/sumatra-pdf-reader-download/9N62S9QBF33F?hl=en-me&gl=me

@kjk can you prove/confirm if "Real eBooks Reader"[ by Xiaoya Labs ](https://apps.microsoft.com/store/detail/real-ebooks-reader/9PK27PJB9FJN)is a blatant breach of SumatraPDF core ? there are as you know many Chinese companies flouting FOSS often offering free, to start or even a few $  then in the case of Enhanced wrappers even charging over 100 $ by aggressive marketing. I am surprised how Artifex tolerate PyMupdf and SumatraPDF on some terms, that they then allow distortion of those terms hitting their own marketing.

**Reply by jgeorgeson** [reply]
I found issue #2567, which links to [this canny roadmap post](https://sumatrapdf.canny.io/feature-requests/p/publish-in-windows-app-store), which says that it's in progress since May 2022, but no updates and I don't find any matches in Windows store searching for "sumatra pdf" or "sumatrapdf".
